### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by qwibbler**
Add tests for all pages checking if:
- [ ] If response status was correct.
- [ ] If a correct template was rendered.
- [ ] If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  it 'show renders correct template' do
+    expect(response).to render_template(:show)
+  end
+
+  it 'shows correct text' do
+    expect(response.body).to include('User page with posts by user')
+  end
+end
```
Comment (Gambit142) [inline spec/controllers/users_controller_spec.rb]: - [x] I know you have a home page in the Pages controller, but you forgot to write requests specs for Users controller, index action, i.e `Users#index`. Kindly write tests for that action in the Users controller. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @qwibbler 👋, 

Good job so far!

## Highlights
- Installed `rspec-rails` gem 💯 
- Wrote requests tests for `Posts` controller 💯 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- In your `README.md` file, kindly include the **_Setup section_**. In your Setup section, kindly inform those who will clone your repository how to clone, bundle install, create the database and run migration files. 👍 

- _Kindly check the comments under the review for other required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: spec/controllers/users_controller_spec.rb (hunk 1 above)
Comment (qwibbler) [reply spec/controllers/users_controller_spec.rb]: What should users#index show? A non-existent page? The homepage?

**Review by andersonlebon — APPROVED** [review]
## Hi @qwibbler 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**